src/transports/types.ts

```diff
@@ -371,6 +375,8 @@ export interface ISegmentParserParsedSegment<T> {
                    number | undefined ]; // end window for the segment
                                          // (part of the segment after that time
                                          // will be ignored)
+  inbandEvents? : IInbandEvent[]; // Inband events parsed from segment data
+  manifestRefreshEvent?: IManifestRefreshEvent; // Inband Manifest refresh events
```

Comment:
Why not just making that one a boolean?
Also saying that this is an event is unnecessary, no?